README.md

```diff
@@ -259,7 +259,7 @@ rosrun rqt_reconfigure rqt_reconfigure
 
 The capture settings available in the `zivid_camera` node matches the settings in the Zivid API.
 To become more familiar with the available settings and what they do, see the API reference for the
-[Settings](http://www.zivid.com/hubfs/softwarefiles/releases/1.2.1+6ca8e224-12/doc/cpp/classZivid_1_1Settings.html)
+[Settings](http://www.zivid.com/hubfs/softwarefiles/releases/1.7.0+a115eaa4-4/doc/cpp/classZivid_1_1Settings.html)
```

Comment:
Do we really not have some URL that always points to the docs for the latest version? This could get annoying when we release so often.

Reply:
We do not have this. I think for now we just have to remember to update these links when we release, especially if we have done changes to Settings. Ideally these links should not go to the API reference but some knowledge base article that describes all the settings, fex https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/99713044/Zivid+One+Camera+Settings, but we don't have good permalinks into the knowledgebase.